test/test_mp3d_eqa.py

```diff
@@ -193,7 +193,7 @@ def test_mp3d_eqa_sim_correspondence():
                 "cur_state.rotation: {} shortest_path.rotation: {} action: {}"
                 "".format(
                     cur_state.position - point.position,
-                    cur_state.rotation - point.rotation,
+                    np.array(cur_state.rotation) - point.rotation,
```

Comment:
This'll need to be something like https://github.com/facebookresearch/habitat-sim/blob/master/habitat_sim/utils.py#L27 instead of just `np.array` as we have been storing quats in `[x, y, z, w]` form instead of the standard `[w, x, y, z]` form because that's how Eigen does it.